schemas/alpha/azure.yaml.json

```diff
@@ -1047,6 +1071,111 @@
             "required": [
                 "deployment"
             ]
+        },
+        "deploymentStacksConfig": {
+            "type": "object",
+            "title": "The deployment stack configuration used for the project.",
+            "additionalProperties": false,
+            "oneOf": [
+                {
+                    "required": [
+                        "bypassStackOutOfSyncError"
+                    ]
+                },
+                {
+                    "required": [
+                        "actionOnUnmanage"
+                    ]
+                },
+                {
+                    "required": [
+                        "denySettings"
+                    ]
+                }
+            ],
+            "properties": {
+                "bypassStackOutOfSyncError": {
```

Comment:
As a follow up to this - it would be good to know what the error experience is today in `azd` when this is set to false and we get a stack out of sync error - I suspect our error behavior today is not as good as it could be and we can polish the error experience here.

Reply:
I'm not exactly sure how to get into this state but it would present as an ARM deployment error.